g2p/apply.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         self.out_g2p_lexicon = self.output_path("g2p.lexicon")
 
     def tasks(self):
-        yield Task("run", mini_task=True)
+        yield Task("run")
```

Comment:
If you have a regular job (no mini_task), you need to define a `self.rqmt` with some reasonable defaults. 
See https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTING.md (we should extend maybe the details in that document)

Reply:
reasonable defaults can/should be task dependent (e.g. on the size of word list) or if that is not always realizable we can expose the rqmt to the user so he can in-/decrease them as needed.
In this case also make sure it does not enter the hash.

Reply:
Can the rqmt enter the hash?
And isn't the rqmt always exposed since it is a public attribute of Job?

Reply:
In the usual way that there is only `self.rqmt = { some defaults }` if will not enter the hash and will be exposed for outside adjustment. I think Willi referred to the case that some jobs that often change requirements have them as actual constructor parameter, and then you need to make sure it is not part of the hash.

Reply:
To use the rqmt for the job:
```suggestion
        yield Task("run", rqmt=self.rqmt)
```